setup.ts

```diff
@@ -247,15 +247,11 @@ async function recaptcha(): Promise<void> {
   }
 }
 async function recaptchaSiteKey(): Promise<void> {
-  console.log(
-    "\nPlease visit this URL to set up reCAPTCHA:\n\nhttps://www.google.com/recaptcha/admin/create"
-  );
-  console.log(
-    '\nSelect reCAPTCHA v2 and the "I`m not a robot" checkbox option'
-  );
-  console.log(
```

Comment:
Remove. I ran the script based on this PR and the text still remains somewhere

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/923cedb7-2c33-4097-b43a-98486fd652ae)